tests/sequential/prometheus_metrics/prometheus_metrics_test.go

```diff
@@ -602,46 +603,14 @@ func testScalerErrors(t *testing.T, data templateData) {
 	KubectlApplyWithTemplate(t, data, "wrongScaledJobTemplate", wrongScaledJobTemplate)
 
 	family := fetchAndParsePrometheusMetrics(t, fmt.Sprintf("curl --insecure %s", kedaOperatorPrometheusURL))
-	val, ok := family["keda_scaler_errors"]
-	assert.True(t, ok, "keda_scaler_errors not available")
-	if ok {
-		errCounterVal1 := getErrorMetricsValue(val)
-
-		// wait for 20 seconds to correctly fetch metrics.
-		time.Sleep(20 * time.Second)
-
-		family = fetchAndParsePrometheusMetrics(t, fmt.Sprintf("curl --insecure %s", kedaOperatorPrometheusURL))
-		val, ok := family["keda_scaler_errors"]
-		assert.True(t, ok, "keda_scaler_errors not available")
-		if ok {
-			errCounterVal2 := getErrorMetricsValue(val)
-			assert.NotEqual(t, errCounterVal2, float64(0))
-			assert.GreaterOrEqual(t, errCounterVal2, errCounterVal1)
-		}
-	}
-	KubectlDeleteWithTemplate(t, data, "wrongScaledJobTemplate", wrongScaledJobTemplate)
-	time.Sleep(2 * time.Second)
-	KubectlApplyWithTemplate(t, data, "scaledJobTemplate", scaledJobTemplate)
-
-	KubectlDeleteWithTemplate(t, data, "wrongScaledObjectTemplate", wrongScaledObjectTemplate)
-	time.Sleep(2 * time.Second)
-	KubectlApplyWithTemplate(t, data, "scaledObjectTemplate", scaledObjectTemplate)
-}
-
-func testScalerErrorsTotal(t *testing.T, data templateData) {
-	t.Log("--- testing scaler errors total ---")
-
-	KubectlDeleteWithTemplate(t, data, "scaledObjectTemplate", scaledObjectTemplate)
-	KubectlApplyWithTemplate(t, data, "wrongScaledObjectTemplate", wrongScaledObjectTemplate)
 
-	family := fetchAndParsePrometheusMetrics(t, fmt.Sprintf("curl --insecure %s", kedaOperatorPrometheusURL))
 	val, ok := family["keda_scaler_errors_total"]
 	assert.True(t, ok, "keda_scaler_errors_total not available")
 	if ok {
 		errCounterVal1 := getErrorMetricsValue(val)
 
-		// wait for 2 seconds as pollinginterval is 2
-		time.Sleep(2 * time.Second)
+		// wait for 20 seconds to correctly fetch metrics.
+		time.Sleep(20 * time.Second)
```

Comment:
was this frequently flaking? it will slow down the test by 18s which is not negligible imho

Reply:
"Wasn't me" 🎤 🎵 https://github.com/kedacore/keda/commit/d593951e9cfdcbe12e98cc6c1ee0d2e7dfb7e729#diff-07e9429b2e4efdad88dbd60082c5c2db53722d506c784add0cdd93938bcc8922R375

Reply:
in that case the PR sgtm, ptal @zroubalik.